src/megaclient.cpp

```diff
@@ -13158,6 +13164,19 @@ handle MegaClient::nextsyncid()
     return currsyncid;
 }
 
+// this method provides a temporal handle useful to indicate putnodes()-local parent linkage
+handle MegaClient::nextUploadId()
+{
+    byte* ptr = (byte*)&mCurrUploadId;
+
+    while (!++*ptr && ptr < (byte*)&mCurrUploadId + NODEHANDLE)
```

Comment:
stopping when the incremented byte turns out to be 0?  Not sure if that is what you meant here - and, if no bytes incremented to 0 then we increment them all, which basically means we would repeat the same ID in only 256 iterations... however, I don't think we need this ID anyway, I'll comment elsewhere about that, thanks